yt/frontends/gamer/cfields.pyx

```diff
@@ -0,0 +1,215 @@
+# distutils: include_dirs = LIB_DIR
+# distutils: libraries = STD_LIBS
+cimport cython
+cimport libc.math as math
+cimport numpy as np
+import numpy as np
+
+np.import_array()
+
+cdef np.float64_t h_eos4(np.float64_t e):
+    cdef np.float64_t x
+    x = 2.25 * e * e
+    return 2.5 * e + x / (1.0 + math.sqrt(x + 1.0))
+
+cdef np.float64_t _lorentz_factor_eos4(
+        np.float64_t rho, 
+        np.float64_t mx,
+        np.float64_t my,
+        np.float64_t mz,
+        np.float64_t e,
+        np.float64_t c
+    ):
+    cdef np.float64_t u2, c2, vx, vy, vz
+    cdef np.float64_t htilde, fac;
+
+    c2 = c * c
+    fac = (1.0 / (rho * (h_eos4(e) + 1.0))) ** 2
+    u2 = (mx * mx + my * my + mz * mz) * fac
+    return math.sqrt(1.0 + u2 / c2)
+
+def gamma_eos4(temp):
```

Comment:
once we're sure it works let's decorate with the boundscheck etc stuff

Reply:
done